src/table/body-cell/index.tsx

```diff
@@ -9,6 +9,7 @@ import { TableTdElement, TableTdElementProps } from './td-element';
 import { InlineEditor } from './inline-editor';
 import LiveRegion from '../../internal/components/live-region/index.js';
```

Comment:
Discarding an edit operation still displays the success icon, which is a little bit confusing.

To reproduce:
1. Start editing a cell
2. Submit the change - you see the success icon
3. Start editing *the same* cell
4. Cancel editing
5. Success icon is still there

Reply:
Fixed.